CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 
 - [#]
+- [#621]: Readme Diagram: Replace duplicate defmt-itm with defmt-rtt
+- [#617]: Add display hint to output `u64` as ISO8601 time
+- [#620]: Tidy up
```

Comment:
nit: I would expand the description of this PR a bit more so one can understand better without reading the PR description. maybe 'remove unused code / dependencies'

Reply:
```suggestion
- [#620]: Tidy up
```
```suggestion
- [#620]: Tidy up: remove unused code and dependencies
```

Reply:
```suggestion
- [#620]: Tidy up: remove unused code and dependencies
```